website/content/docs/connect/ca/index.mdx

```diff
@@ -4,6 +4,26 @@ page_title: Connect - Certificate Management
 description: An overview of the Connect Certificate Authority mechanisms.
 ---
 
+### CA and Certificate relationship
+
+This diagram shows the relationship between the CA certificates in Consul primary and
+secondary.
+
+![CA relationship](/img/cert-relationship.svg)
+
+In the primary datacenter, the Consul and AWS providers use the Primary Root CA to sign
+leaf certificates. The Vault provider uses an intermediate CA to sign leaf certificates.
```

Comment:
Where is AWS coming from? Can we make this generic and avoid specifying specific cloud providers? Same goes for Vault. Introducing AWS and Vault seems a bit too specific for an overview.

Reply:
yes right, I will remove this from here and we can add it in a separate section later.

Reply:
@dhiaayachi  You know, we could just use `DC1` and `DC2` that is a pattern we use throughout the docs. That would address the problem. Plus, you did specify Consul primary and secondary datacenter in the text.